.github/workflows/python_collect_docstrings.yml

```diff
@@ -0,0 +1,39 @@
+name: pyfiction Collect Docstrings
+
+on: workflow_dispatch
+
+jobs:  
+  get_docstrings:    
+    name: collect docstrings    
+    runs-on: macos-latest  
+    steps:      
+      - name: Setup Python        
+        uses: actions/setup-python@v4        
+        with:          
+          python-version: "3.11.x"      
+
+      - name: Clone Repository        
+        uses: actions/checkout@v3        
+        with:          
+          submodules: recursive            
+
+      - name: Install pybind11_mkdoc  
+        working-directory: ${{github.workspace}}        
+        run: pip install pybind11_mkdoc        
+
+      - name: Install clang        
+        working-directory: ${{github.workspace}}        
+        run: pip install clang==14            
+
+      - name: create docstring file
+        working-directory: ${{github.workspace}}  
+        run: touch ${{github.workspace}}/docstrings.h
+
+      - name: mkdoc        
+        working-directory: ${{github.workspace}}        
+        run: python3 -m pybind11_mkdoc -o ${{github.workspace}}/docstrings.h `find ${{github.workspace}}/include/fiction -name "*.hpp" -print`      
```

Comment:
Since we use the `.hpp` header extension, can `docstrings` please also use it for consistency?